src/pages/Graph/GraphPage.tsx

```diff
@@ -302,30 +306,28 @@ export class GraphPage extends React.Component<GraphPageProps> {
     return (
       <>
         <FlexView className={conStyle} column={true}>
-          <div>
-            <Breadcrumb title={true}>
-              <Breadcrumb.Item active={true}>
-                {this.props.node && this.props.node.nodeType !== NodeType.UNKNOWN
-                  ? `Graph for ${this.props.node.nodeType}: ${this.getTitle(this.props.node)}`
-                  : 'Graph'}
-                <OverlayTrigger
-                  key={'graph-tour-help-ot'}
-                  placement="right"
-                  overlay={<Tooltip id={'graph-tour-help-tt'}>Graph help tour...</Tooltip>}
-                >
-                  <Button bsStyle="link" style={{ paddingLeft: '6px' }} onClick={this.toggleHelp}>
+          <div className={breadcrumbStyle}>
+            <Breadcrumb>
+              <BreadcrumbItem isActive={true}>
+                <Title headingLevel="h4" size="xl">
+                  {this.props.node && this.props.node.nodeType !== NodeType.UNKNOWN
+                    ? `Graph for ${this.props.node.nodeType}: ${this.getTitle(this.props.node)}`
+                    : 'Graph'}
+                </Title>
+                <Tooltip key={'graph-tour-help-ot'} position={TooltipPosition.right} content="Graph help tour...">
+                  <Button variant="link" style={{ paddingLeft: '6px' }} onClick={this.toggleHelp}>
                     <KialiIcon.Help className={defaultIconStyle} />
                   </Button>
-                </OverlayTrigger>
-              </Breadcrumb.Item>
-              {this.props.graphTimestamp > 0 && (
-                <span className={'pull-right'}>
-                  {new Date(graphStart).toLocaleDateString(undefined, timeDisplayOptions)}
-                  {' ... '}
-                  {new Date(graphEnd).toLocaleDateString(undefined, timeDisplayOptions)}
-                </span>
-              )}
+                </Tooltip>
+              </BreadcrumbItem>
             </Breadcrumb>
+            {this.props.graphTimestamp > 0 && (
+              <span className={'pull-right'}>
+                {new Date(graphStart).toLocaleDateString(undefined, timeDisplayOptions)}
+                {' ... '}
+                {new Date(graphEnd).toLocaleDateString(undefined, timeDisplayOptions)}
+              </span>
+            )}
           </div>
           <div>
             {/* Use empty div to reset the flex, this component doesn't seem to like that. It renders all its contents in the center */}
```

Comment:
Can you remove empty div?

Reply:
I'm not sure where is the empty div. But I removed the comment, because looks obsolete.